dependencies/example-projects/grpc-vs-google-api-client/README.md

```diff
@@ -0,0 +1,62 @@
+# grpc 1.17.1 v.s. google-api-client 1.27.0
```

Comment:
```suggestion
# grpc 1.17.1 vs google-api-client 1.27.0
```

Let's be consistent. "vs" is okay, as is "vs." or "versus".

Reply:
Updated to "vs".